framework/decode/vulkan_replay_consumer_base.h

```diff
@@ -1457,7 +1457,7 @@ class VulkanReplayConsumerBase : public VulkanConsumer
 
     const VulkanReplayOptions& options_;
 
-    VulkanReplayDumpResources* resource_dumper_{ nullptr };
+    std::unique_ptr<VulkanReplayDumpResources> resource_dumper_{ nullptr };
```

Comment:
default for a `unique_ptr` is no pointer, e.g. nullptr, so you don't need an initializer here